### PR TITLE
T6619: Remove the remaining uses of per-protocol FRR configs

### PR DESCRIPTION
**Pull request by HollyGurza**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [ ] Other (please describe):

## Related Task(s)

* https://vyos.dev/T6619
* https://vyos.dev/T6623

## How to test

```
conf
set protocols static multicast route 224.202.0.0/24 next-hop 224.203.0.1
set protocols static multicast interface-route 224.203.0.0/24 next-hop-interface eth0 
commit
vtysh -c 'show running-config'
```

## Smoketest result

```
vyos@vyos:~$ python3 /usr/libexec/vyos/tests/smoke/cli/test_protocols_static_multicast.py 
test_01_static_multicast (__main__.TestProtocolsStaticMulticast.test_01_static_multicast) ... ok

----------------------------------------------------------------------
Ran 1 test in 6.213s

OK
```

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [x] I have linked this PR to one or more Phabricator Task(s)
- [x] I have run the components [**SMOKETESTS**](https://github.com/vyos/vyos-1x/tree/current/smoketest/scripts/cli) if applicable
- [x] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: smoketest/scripts/cli/test_protocols_static_multicast.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+        # Verify FRR bgpd configuration
+        frrconfig = self.getFRRconfig('ip mroute', end='')
+
+        self.assertIn('ip mroute 224.202.0.0/24 224.203.0.1', frrconfig)
+        self.assertIn('ip mroute 224.203.0.0/24 eth0', frrconfig)
+
+
+if __name__ == '__main__':
+    unittest.main(verbosity=2, failfast=True)
```
Comment (c-po) [inline smoketest/scripts/cli/test_protocols_static_multicast.py]: Please drop `failfast` here

**Review by c-po — APPROVED** [review]

**Comment by c-po** [comment]
@MergifyIo backport circinus

File: smoketest/scripts/cli/test_protocols_static_multicast.py (hunk 1 above)
Comment (HollyGurza) [reply smoketest/scripts/cli/test_protocols_static_multicast.py]: done

**Review by dmbaturin — APPROVED** [review]

**Review by sever-sever — APPROVED** [review]

**Comment by sever-sever** [comment]
@MergifyIo backport sagitta

**Pull request merged**